src/app/widgets/Visualizer/index.styl

```diff
@@ -91,7 +91,34 @@
     }
 }
 
+.control-area {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    top: 20px;
+    right: 0;
+    left: 0;
+}
+
+.status-message {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-self: center;
+    justify-self: center;
+    font-size: clamp(1rem, 1.15vw, 2rem);
+    height: calc(var(--vh, 1vh) * 10);
+    max-height: 70px;
+    border-radius: 5px;
+    width: 100%;
+    max-width: 255px;
+    text-align: center;
 
+    background-color: rgba(rgb(244, 134, 6), 1);
```

Comment:
I'd add some opacity to the control background to remain consistent with other parts of the Machine Status area.
I'd also look at line-height since there's some overlapping on higher width resolutions
![machine_status](https://user-images.githubusercontent.com/22222507/179523530-15085b8f-abcb-4354-b3a3-a5d3179a0a3a.png)